nasa/openmct
# [Imagery Plugin] Enhancements - Compass, HUD, Freshness

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | In Progress
Command line build passes? | Not yet
Changes have been smoke-tested? | Yes
Testing instructions included? | Yes

Closes #3665 (main setup testing instructions and imagery freshness instructions)
Closes #3658 (compass/hud testing instructions, make sure main setup instructions are followed above)
